### DISCUSSION TITLE
is it possible to get a duffel bag?

### DISCUSSION THREAD

**Discussion by snwyyy** [YimMenu/YimMenu#1473] [Q&A] [answered]
i just dont know how the outfit editor works so maybe anyone can help and if its even possible to get a duffel bag

**Comment by SlavicBlin** [comment, marked as answer]
yes it is possible, if you go to outfit editor and then hand type the numbers,  81 ( Open Duffle ),  82 ( Closed Duffle ), 85 ( Open Designer Duffle ), 86 ( Closed Designer Duffle ) and then you can also edit the color / design of it.
![image](https://github.com/YimMenu/YimMenu/assets/95110595/bcaa4edc-c346-4c47-aaf8-ecfa47dc731a)

**Reply by snwyyy** [reply]
also another question is it detectable?

**Reply by SlavicBlin** [reply]
The game might remove the outfit if saved, but otherwise no its not 👍

**Reply by snwyyy** [reply]
ty also is there a way to get white joggers?

**Reply by SlavicBlin** [reply]
yes there is you just have to find the numbers for it

**Reply by snwyyy** [reply]
alr
